platform-sdk/docs/proposals/proposal-process.md

```diff
@@ -0,0 +1,119 @@
+but not merged into the source code of the platform.
+
+While in this draft phase the proposal may be updated and modified as needed. It is best practice to solicit review from
+stakeholders to surface any potential change requests while the proposal is in the draft phase. All grammar and spelling
+errors should be caught during this phase.
+
+### Creating a Draft Proposal
+
+1. Create an issue in the github repository for the creation of the proposal if it is for a project prioritized by
+   SwirldsLabs. Other proposals do not require a ticket.
+2. Create a new branch for the proposal.
+3. Copy the template `template.md` into a subdirectory of `platform-sdk/docs/proposals/` with the name of the
+   proposal.
+4. Fill out the template with the details of the proposal, removing any irrelevant sections. Diagrams and other files
+   may be included in the proposal directory and linked from the proposal.
+5. Create a draft PR with the proposal documents.
+6. Put the draft PR in the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board in
+   the `Draft` status.
+7. Solicit feedback from stakeholders and other design collaborators.
+8. Iterate on the proposal until it is ready for voting. All open comments should be resolved.
+
+### Submitting a Proposal for Voting
+
+When the proposal is ready for voting, take the following steps:
+
+1. Ensure all comments are resolved.
+9. Mark the as ready for review
+10. Update the status of the PR in
+    the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board to `Voting`
+
+Requests for inmaterial changes to the proposal can be addressed by pushing additional commits to the existing proposal
+PR while in `Voting`. Examples of inmaterial changes are spelling or grammar errors, wording changes that do not modify
+meaning or intent, and formatting changes. Examples of material changes include changes to behaviors or APIs, addition
+or removal of content, etc. If any material changes are needed, the PR must be closed, and moved to the `Superseded`
+status. A new proposal PR is prepared and voting is restarted. The `Superseded` PR must reference the new PR.
+
+---
+
+## Voting on A Proposal
+
+Votes follow the [Apache Voting](https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions)
+scheme. Votes are cast in comments on the PR on a range between -1 to -0, +0 to +1 with the following semantics:
+
+* -1: is a veto.
+* -0 >= x > -1: is a vote generally against the proposal.
+* +0 <= x < +1: is a vote generally in favor of the proposal.
+* +1: is a vote in favor of the proposal.
+
+The magnitude of the numeric value of the vote indicates the strength of the sentiment behind the vote.
+
++1 votes can also be expressed by approving the PR. -1 votes can be expressed by "Requesting Changes" on the PR and
+must be accompanied by a comment explaining the reason for the veto.
```

Comment:
I miss a "please give me more time" option. Let's assume I really want to give feedback to a proposal but currently do not have time to do a deep review. Doing "-1" here seams hard but it is currently the only solution to not get a proposal accepted for that you want to add your thoughts in future.

Reply:
If the minimum threshold of +1s is reached, then it should be allowed to go forward. We do need to impress upon people the gravity of voting +1 and -1. You should only vote either way if you really understand and grasp the proposal, and think it's a great idea that we should implement.